app/jest.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
     '^@clinics/*': '<rootDir>/public/clinic-output/clinics-zip-code-lookup$1',
     '^@components(.*)$': '<rootDir>/src/components$1',
     '^@context(.*)$': '<rootDir>/src/context$1',
+    '^@customTypes(.*)$': '<rootDir>/src/customTypes$1',
```

Comment:
@karinamzalez I could use your help with this. 

- I want to declare some custom types. This will get us started on https://wicmtdp.atlassian.net/browse/WMDP-254
- I want to use one of these nice path aliases you set up for us
- I can't use `@types` because that's a reserved term in Typescript

What's the best practice for doing this? I assume it's not `@customTypes` 😅 but that's what I put in temporarily. Could you do some research into this?

Reply:
Talked to @karinamzalez: going to try `@sharedTypes`